pkg/tfgen/docs_test.go

```diff
@@ -504,3 +505,36 @@ FooFactory fooFactory = new FooFactory();
 
 	assert.Equal(t, buf.String(), hclConversionsToString(input))
 }
+
+func TestGroupLines(t *testing.T) {
```

Comment:
love this but unsure if it needs to be in this PR?

Reply:
I had to understand how it worked in order to do this PR. It's equivalent to adding documentation.